tasks.py

```diff
@@ -38,7 +44,7 @@ def is_truthy(arg):
 namespace.configure(
     {
         "welcome_wizard": {
-            "nautobot_ver": "latest",
+            "nautobot_ver": "1.5.5",
```

Comment:
Probably should be at least 1.6.0 here?

Reply:
1.6 drops support for python 3.7. In order to keep the existing Pyproject.toml, I changed this to 1.5.5. I tried with 1.6.0, but I get dependency resolution issues in the docker build stage.

Reply:
Are we still supposed to be supporting python 3.7 for this project? It was EOL a year ago.

Reply:
I don't believe we will be creating another release of Welcome Wizard for 1.6. I was trying to make the minimum number of changes required to get Upstream Testing to work.

Reply:
I retested, I got this to work with 1.6, which pulls the latest 1.6.x branch. It is possible that 1.6.0 didn't work, or something else changed as well.